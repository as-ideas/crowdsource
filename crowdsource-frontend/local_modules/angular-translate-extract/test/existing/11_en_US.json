{
    "COMMON": {
        "A_VIRTUAL_TRANSLATION": "A virtual translation",
        "A_TRANSLATION": "Just a translation",
        "A_VIRTUAL_LINK": "@:COMMON.A_VIRTUAL_TRANSLATION",
        "A_TRUE_LINK": "@:COMMON.A_TRANSLATION"
    }
}
