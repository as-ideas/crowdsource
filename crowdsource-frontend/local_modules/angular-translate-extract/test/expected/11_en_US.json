{
    "COMMON": {
        "A_VIRTUAL_TRANSLATION": "A virtual translation",
        "A_VIRTUAL_LINK": "@:COMMON.A_VIRTUAL_TRANSLATION",
        "A_TRANSLATION": "Just a translation",
        "A_TRUE_LINK": "@:COMMON.A_TRANSLATION"
    }
}
